packages/utilities/src/EnvFileUtils.ts

```diff
@@ -0,0 +1,107 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { existsSync, readFileSync } from "fs";
+import { homedir } from "os";
+import { join } from "path";
+import { ImperativeError } from "../../error/src/ImperativeError";
+import { EnvironmentalVariableSettings } from "../../imperative/src/env/EnvironmentalVariableSettings";
+import * as JSONC from "comment-json";
+
+/**
+ * Utility to load environment JSON files and set variables
+ * @export
+ * @class EnvFileUtils
+ */
+export class EnvFileUtils {
+
+    /**
+     * Check and read in an environment file from the user home directory using the app name
+     * If the file is valid, set the environment variables
+     * If the file is not valid, display an error and continue
+     * @param {string} appName - The application name
+     * @param {boolean} checkCliHomeVariableFirst - Check inside of *_CLI_HOME first if it is defined
+     * @param {string} envPrefix - The environment variable prefix
+     * @returns {void}
+     * @throws {ImperativeError}
+     */
+    public static setEnvironmentForApp(appName: string, checkCliHomeVariableFirst: boolean = false, envPrefix?: string) {
```

Comment:
should we be explicit about the return type (`... : void`)?
Also, the `:boolean` type definition might be redundant 😋